tests/unit/test_plotting/test_quick_plot.py

```diff
@@ -5,9 +5,14 @@
 import numpy as np
 from tempfile import TemporaryDirectory
 
+_solver_args = [pybamm.CasadiSolver()]
+if pybamm.has_idaklu():
+    _solver_args.append(pybamm.IDAKLUSolver())
+
```

Comment:
Extremely minor, but can this be moved into the `TestQuickPlot` class so that it is not a global in the file?

Something like this could be used to avoid having to make an init function.
```python
class TestQuickPlot:
    _solver_args = [pybamm.CasadiSolver()] + 
                   [pybamm.IDAKLUSolver()] if pybamm.has_idaklu() else []
```
Though I am sure pre-commit will change that awkward style